tests/unit/Assets/Filters/CssMinTest.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Phalcon\Test\Unit\Assets\Filters;
+
+use Phalcon\Assets\Resource;
+use Phalcon\Test\Proxy\Assets\Filters\Cssmin;
+use Phalcon\Test\Module\UnitTest;
+
+/**
+ * \Phalcon\Test\Unit\Assets\Filters\Cssmin.php
+ * Tests the \Phalcon\Assets\Filters\Cssmin component
+ *
+ * @copyright (c) 2011-2016 Phalcon Team
+ * @link      http://www.phalconphp.com
+ * @author    Andres Gutierrez <andres@phalconphp.com>
+ * @author    Nikolaos Dimopoulos <nikos@phalconphp.com>
+ * @package   Phalcon\Test\Unit\Assets\Filters
+ * @group     assets
+ *
+ * The contents of this file are subject to the New BSD License that is
+ * bundled with this package in the file docs/LICENSE.txt
+ *
+ * If you did not receive a copy of the license and are unable to obtain it
+ * through the world-wide-web, please send an email to license@phalconphp.com
+ * so that we can send you a copy immediately.
+ */
+class CssMinTest extends UnitTest
+{
+    /**
+     * Tests cssmin filter with empty parameters
+     *
+     * @author Nikolaos Dimopoulos <nikos@phalconphp.com>
+     * @since  2014-10-10
+     */
+    public function testAssetsFilterCssmin()
+    {
+        $this->specify(
+            "The cssmin filter needs a string parameter",
+            function () {
+                $cssmin = new Cssmin();
+
+                try {
+                    $cssmin->filter(new \stdClass());
+                } catch (\InvalidArgumentException $e) {
+                    $expected = "Parameter 'content' must be a string";
+                    expect($e->getMessage())->equals($expected);
+                }
+            }
+        );
+
+        $this->specify(
+            "The cssmin filter with empty parameter does not return empty back",
+            function () {
+
+                $cssmin = new Cssmin();
+                $actual = $cssmin->filter('');
+                expect($actual)->isEmpty();
+            }
+        );
+
+        $this->specify(
+            "The cssmin filter with space as parameter does not return correct results",
+            function () {
+
+                $cssmin   = new Cssmin();
+                $actual   = $cssmin->filter(' ');
+                $expected = ' ';
+                expect($actual)->equals($expected);
+            }
+        );
+
+        $this->specify(
+            "The cssmin filter with brackets does not return correct results",
+            function () {
+
+                $cssmin   = new Cssmin();
+                $actual   = $cssmin->filter('{}}');
+                $expected = '{}}';
+                expect($actual)->equals($expected);
+            }
+        );
+
+        $this->specify(
+            "The cssmin filter with brackets and spaces does not return correct results",
+            function () {
+
+                $cssmin   = new Cssmin();
+                $actual   = $cssmin->filter('.s { d     :        b; }');
+                $expected = '.s{d : b;}';
+                expect($actual)->equals($expected);
```

Comment:
expect->equals was created for not to write what is actual and what is expected.
So this can be a one-liner:

```php
expect('$cssmin->filter('.s { d     :        b; }')->equals('.s{d : b;}');
```

expect equals expected sounds ugly to me :)